packages/features/bookings/Booker/Booker.tsx

```diff
@@ -77,7 +86,9 @@ const BookerComponent = ({ username, eventSlug, month, rescheduleBooking }: Book
     if (layout === "mobile") {
       timeslotsRef.current?.scrollIntoView({ behavior: "smooth" });
     }
-  }, [layout, selectedDate]);
+
+    LargeLayouts.includes(layout) && !selectedDate && setSelectedDate(dayjs().format("YYYY-MM-DD"));
```

Comment:
Would it make sense to control this behaviour in the booker store? (as part of the setlayout function) Because that it's really tied into changing layouts, and will also work if there's another place where we would start updating layouts, because all of them have to fire this method.

Reply:
Added it in this commit; https://github.com/calcom/cal.com/pull/8484/commits/0296c26bc9955e3eabb3430dda69b0b3ca471d34